### ISSUE TITLE
Add renaming of normalization variables for fine-tuning

### ISSUE THREAD

**Issue by jpdunc23** [ai2cm/ace#461]
https://github.com/ai2cm/full-model/pull/1200 added a renaming feature as part of `XarrayDataConfig` which is helpful in the double call case. This feature could also be used for fine-tuning where the stepper's names and the dataset's names differ, but to do so we would also need to rename the normalization (see the conversation in https://github.com/ai2cm/full-model/pull/1207.

One possibility to do so is add a new renaming feature to the stepper, which would have the added benefit of avoiding repetition of `rename_variables` in both `train_loader` and `validation_loader`.
